docs/sources/salesforce.md

```diff
@@ -20,7 +20,7 @@ We will be using `OpenSSL` but any other certificate generation tool should work
     openssl req -x509 -sha256 -nodes -days 36500 -newkey rsa:2048 -keyout tm-sf.key -out tm-sf.crt
     ```
 
-1. At Salesforce site select `Setup > Apps > App Manager`, click on `New Connected App`.
+2. At Salesforce site select `Setup > Apps > App Manager`, click on `New Connected App`.
```

Comment:
For the same reason as explained in my previous comment, the numbering was correct, and this change should be reverted.